java/mtsj/core/src/main/java/com/devonfw/application/mtsj/mailservice/logic/impl/MailMock.java

```diff
@@ -17,8 +16,9 @@
 @ConditionalOnProperty(prefix = "spring.mail", name = "enabled", havingValue = "false")
 public class MailMock implements Mail {
 
-  @Value("${spring.mail.username}")
-  private String from;
+  /*
+   * @Value("${spring.mail.username}") private String from;
```

Comment:
Samehere. Actually looks quite redundant. Introduce common abstract class `AbstractMail` inherited from `MailImpl` and `MailMock`.
BTW: The name `Mail` is kind of odd and confusing. This is not a Mail/Email but a component that can send Emails. Hence it should be renamed. Actually this is a use-case in the logic-layer. It should therefore be `Mailmanagement` (with consistent package rename from `mailservice` or the interface should also be named `Mailservice`).